modules/presets/enable-tor.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, ... }:
+let
+  defaultTrue = lib.mkDefault true;
+in {
+  services.tor = {
+    enable = true;
+    client.enable = true;
+  };
+
+  # Use Tor for all outgoing connections
+  services = {
+    bitcoind.enforceTor = true;
+    clightning.enforceTor = true;
+    lnd.enforceTor = true;
+    lightning-loop.enforceTor = true;
+    liquidd.enforceTor = true;
+    electrs.enforceTor = true;
+    # disable Tor enforcement until btcpayserver can fetch rates over Tor
+    # btcpayserver.enforceTor = true;
+    nbxplorer.enforceTor = true;
+    spark-wallet.enforceTor = true;
+    recurring-donations.enforceTor = true;
+  };
+
+  # Add onion services for incoming connections
+  nix-bitcoin.onionServices = {
+    bitcoind.enable = defaultTrue;
+    clightning.enable = defaultTrue;
+    lnd.enable = defaultTrue;
```

Comment:
Agreed. I had a fix queued up in another (unreleased PR) as I wanted to limit the scope of this PR, but I've added this now (`fixup! disable default onion services for clightning, lnd, btcpayserver`). 

The default onion service for btcpayserver is a downright security risk because the webinterface allows any visitor to register an admin account on a freshly setup node. (docs: `The first created account on a newly-deployed BTCPay Server is automatically admin.`)